src/components/AmountTextField.jsx

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 
 const AmountTextField = (props) => (
   <TextField
-    sx={{ maxWidth: 130 }}
+    sx={{ maxWidth: 120 }}
     variant="outlined"
     placeholder="Unknown"
```

Comment:
Instead of always using `Unknown` how about accepting an optional `placeholder` so this can be something like "0" or "Full Tuition" when the filter uses this?

Reply:
when would we display `full tuition`?
On the figma design it shows it when the max input is empty but that might be confusing?